definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-bailiff-nonprod.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "LiveFrom": "01/02/2021",
```

Comment:
not 24/02?

Reply:
I know it doesn't matter. It's more like for humans who will read it to see what was added when without blaming file.